src/docs/sphinx/tutorials/step12/Tutorial.rst

```diff
@@ -0,0 +1,358 @@
+.. _TutorialPoroPlasticity:
+
+
+####################################################
+Tutorial 12: PoroMechanical Models for Wellbore Problems
+####################################################
+
+
+**Context**
+
+The main objective of this tutorial is to demonstrate how to use internal mesh generator and poromechanical solvers in GEOSX to tackle wellbore problems in porous media. In this tutorial, a poroplastic model is applied to find the solution of rock deformation within the vicinity of a vertical wellbore, considering elastoplastic deformation, fluid diffusion and poromechanical coupling effect. To do so, a single phase flow solver is fully coupled with a small-strain Lagrangian mechanics solver in GEOSX and the Extended Drucker-Prager model (see :ref:`TwoInvariantPlasticity`) is chosen as the material model for the solid domain. We first solve this problem with the existing poroelastic solver and verify the modeling results with the corresponding analytical solutions. Then, the verified case is employed to test the newly implemented poroplasticity solver in GEOSX, whose results are compared with the ones obtained from the poroelastic case to highlight the impact of plasticity in this specific problem.
```

Comment:
What is added to this tutorial comparing to the 11st?

Reply:
This tutorial focuses on the poromechanical solvers (poroelastic and poroplastic) and coupling effect, which is not covered in the 11th one.